.travis.yml

```diff
@@ -2,26 +2,27 @@ language: php
 
 sudo: true
 
+addons:
+  apt:
+    packages:
+      - oracle-java8-set-default
+
 matrix:
   fast_finish: true
   include:
     - php: 5.6
     - php: 7.0
     - php: 7.1
-      env: ES_VERSION="2.4.4"
-    - php: 7.1
-      env: ES_VERSION="1.7.6" DOWNLOAD_URL="https://download.elastic.co/elasticsearch/elasticsearch/elasticsearch-${ES_VERSION}.deb"
     - php: hhvm
   allow_failures:
     - php: hhvm
 
 env:
   global:
-    - ES_VERSION="2.4.4"
-    - DOWNLOAD_URL="https://download.elastic.co/elasticsearch/release/org/elasticsearch/distribution/deb/elasticsearch/${ES_VERSION}/elasticsearch-${ES_VERSION}.deb"
+    - ES_VERSION="5.2.2"
 
 before_install:
-  - curl -O $DOWNLOAD_URL && sudo dpkg -i --force-confnew elasticsearch-${ES_VERSION}.deb && sudo service elasticsearch restart
+  - curl -O "https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-${ES_VERSION}.deb" && sudo dpkg -i --force-confnew elasticsearch-${ES_VERSION}.deb && sudo service elasticsearch restart
```

Comment:
nit: split in two lines? I think travis already exits if one of these lines fails?